components/hab/src/main.rs

```diff
@@ -15,7 +15,8 @@ use hab::{cli::{self,
           command::{self,
                     pkg::{download::{PackageSet,
                                      PackageSetFile},
-                          list::ListingType}},
+                          list::ListingType,
+                          uninstall_impl::UninstallHookMode}},
```

Comment:
Yeah, `uninstall_impl` should probably be a module of `uninstall` instead of them residing at the same level like they do now. And `uninstall` should probably `pub use` `UninstallHookMode` and `UninstallSafety`.